enhancements/machine-config/manage-boot-images.md

```diff
@@ -0,0 +1,197 @@
+---
+title: manage-boot-images
+authors:
+  - "@djoshy"
+reviewers: 
+  - "@yuqi-zhang"
+  - "@mrunal"
+  - "@cgwalters, for rhcos context" 
+  - "@joelspeed, for machine-api context" 
+  - "@sdodson, for installer context"
+approvers:
+  - "@yuqi-zhang"
+api-approvers: 
+  - "@joelspeed" 
+  - "@murnal" 
+creation-date: 2023-10-05
+last-updated: 2022-10-05
+tracking-link:
+  - https://issues.redhat.com/browse/MCO-589
+see-also:
+replaces:
+superseded-by: https://github.com/openshift/enhancements/pull/201, https://github.com/openshift/enhancements/pull/368
```

Comment:
Shouldn't this be under `replaces`?

Reply:
Will fix, missed that!

Reply:
Actually - I want to clarify here. After giving it some thought: this enhancement replaces #368, but supersedes #201 as it uses work done from there(I guess #201 could also be under see-also). Is that what you meant? Not strongly held

Reply:
Makes sense to me!